explorer/client/src/pages/object-result/views/shared/ObjHeader.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { ReactComponent as ObjIcon } from '../../../../assets/SVGIcons/Call.svg';
+import GoBack from '../../../../components/goback/GoBack';
+import Longtext from '../../../../components/longtext/Longtext';
+
+import styles from './ObjHeader.module.css';
+
+type ObjHeaderData = {
+    objId: string;
+    objKind: 'Object' | 'Package';
+    objName?: string;
+};
+
+function ObjAddressHeader({ data }: { data: ObjHeaderData }) {
+    const ObjectKindName = data.objKind;
```

Comment:
Why extract this here instead of inline like the rest of the props?

Reply:
This was originally for consistency with `src/pages/transaction-result/TxResultHeader.tsx` but I agree.